test/integration/targets/apt_virtual_pkg_fix/tasks/main.yml

```diff
@@ -0,0 +1,58 @@
+---
```

Comment:
Rather than creating a new `apt_virtual_pkg_fix` role, 
1. Move this under `test/integration/targets/apt/tasks/apt_virtual_pkg_fix.yml`.
2. Do `import_tasks: apt_virtual_pkg_fix.yml` in `test/integration/targets/apt/tasks/main.yml`